src/App.tsx

```diff
@@ -1,61 +1,153 @@
+import { useState } from 'react';
+
 import './App.scss';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { TodoList } from './components/TodoList';
+import { PreparedTodo, Todo } from './types';
+
+function getUserById(userId: number) {
+  return usersFromServer.find((user) => user.id === userId) || null;
```

Comment:
@VeDono By default, the `find` method returns `undefined` if it does not find anything, so it is not necessary to specify `null`, you can simply specify an additional type of `undefined` for the `user` interface
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find